tests/test_sharded_zarr_store.py

```
@@ -0,0 +1,669 @@
+import asyncio
```

Ruff (0.11.9)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)